.github/workflows/deploy.yml

```diff
@@ -4,7 +4,7 @@ name: 'deploy'
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Is there anything that should be configured on `dokku/github-action`, or will it automatically push the branch that triggered the action (`main`) to remote `master`?

Reply:
It automatically pushes the branch that triggered the workflow. I should be able to verify that once this is merged.